benchmark/linear_attention/benchmark_single_linear_attention.py

```
@@ -548,10 +626,26 @@ def flash_qla_linear_attention(query, key, value, gate, beta, write_gate, s0):
             except ImportError as e:
                 raise RuntimeError(f"The installed fla does not provide KDA (fla.ops.kda): {e}")
         else:  # gdn2
-            raise ValueError("gdn2 is only supported with the 'cudnn' backend (no fla implementation, so no reference either); use --skip_ref")
+            try:
+                from fla.ops.gdn2 import chunk_gdn2
+            except ImportError as e:
+                raise RuntimeError(f"The installed fla does not provide GDN2 (fla.ops.gdn2): {e}")
```

Ruff (0.16.1)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)